Test.C

```diff
@@ -150,14 +150,35 @@ int main(){
 	      while (encryptionPointer != null){
             // details about the encryption type itself is stored in encryptionStock
             // amount of this encryption type that the player has is stored in encryptionInventory
-            printf("%s\n", encryptionPointer->encryption->name);
-            encryptionPointer=encryptionPointer->next;
-        }  //current encrypt
+            printf("%s\n", encryptionPointer->encryption->name); //current encrypt
+            encryptionPointer=encryptionPointer->next; //pointer
+        } //end while loop
+		
         //iterate and print through current encryption
-
-        //receive credits NATHAN
-        //use equation to figure out how many credit user receives
+    }
+        
+    //receive credits
+		int sum;
+		int newCredits;
+		printf("Original credits: %d\n", player.currentCredits); //prints out the current credits the player has
+
+    //use equation to figure out how many credit user receives
+		newCredits = player.currentCredits * Encryption.effectiveness * Equipment.messageAmount;
```

Comment:
```suggestion

```
Seems like this was left as pseudocode but you have it implemented below so we don't need it

Reply:
yeah, this was what we talked about before we did all that stuff on the zoom call